specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/Dataset.json

```diff
@@ -1078,13 +1326,92 @@
     },
     "CommonDataServiceForAppsEntityDatasetTypeProperties": {
       "description": "Common Data Service for Apps entity dataset properties.",
+      "type": "object",
       "properties": {
         "entityName": {
           "type": "object",
           "description": "The logical name of the entity. Type: string (or Expression with resultType string)."
         }
       }
     },
+    "AzureDataLakeStoreDataset": {
+      "x-ms-discriminator-value": "AzureDataLakeStoreFile",
+      "description": "Azure Data Lake Store dataset.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Dataset"
+        }
+      ],
+      "properties": {
+        "typeProperties": {
+          "description": "Azure Data Lake Store dataset properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AzureDataLakeStoreDatasetTypeProperties"
+        }
+      }
+    },
+    "AzureDataLakeStoreDatasetTypeProperties": {
+      "description": "Azure Data Lake Store dataset properties.",
+      "type": "object",
+      "properties": {
+        "folderPath": {
+          "type": "object",
```

Comment:
That seems inconsistent with the description that says it's a string

Reply:
@drosales57 I also think string would be a better choice here. Is it possible to use string?